src/DataTable/filters/MultiSelectDropdownFilter.jsx

```diff
@@ -21,20 +21,26 @@ function MultiSelectDropdownFilter({
     checkedBoxes.push(value);
     return setFilter(checkedBoxes);
   };
-  const headerBasedId = useMemo(() => `checkbox-filter-check-${getHeaderProps().key}-`, [getHeaderProps]);
+
   return (
     <DropdownButton variant="outline-primary" id={ariaLabel.current} title={Header}>
-      <div role="group" aria-label={Header} className="pgn__dropdown-filter-checkbox-group">
```

Comment:
[clarification] Is it intentional to remove the `role="group"` attribute here?

Reply:
Yes, Form.CheckboxSet have `role ="group"` by default

<img width="531" alt="image" src="https://github.com/openedx/paragon/assets/93188219/e7fb586a-03b7-47d3-bbe3-f346d166da0a">

Reply:
Thanks for confirming! 🎉